install/update_94_95.php

```diff
@@ -1784,7 +1784,15 @@ function update94to95() {
       );
    }
 
+   // Add values planning_days in glpi_config to manage displayed days in new timeline view
+   if ($DB->tableExists('glpi_configs')) {
```

Comment:
This is useless, this table do exists. Also, please avoid raw SQL (nad I'm pretty sure there is a method to quickly add new entries for config in migrations)

Reply:
corrected